.github/workflows/ci.yml

```diff
@@ -42,6 +42,7 @@ jobs:
           key: swig-${{ matrix.os }}-v4.1.1
```

Comment:
Should we use a variable for the `SWIG_VERSION` too, while we're changing things?

Reply:
I'd rather not if we don't have to right now. I don't enjoy wrangling CI config (as you can tell from my suggestion to drop caching rather than fixing it :) ).

Reply:
Haha, no worries :)